CHANGELOG.md

```diff
@@ -23,13 +23,19 @@ significant modifications will be credited to OpenTelemetry Authors.
 ([#141](https://github.com/open-telemetry/opentelemetry-demo-webstore/pull/141))
 * Added additional attributes to productcatalog service
 ([#143](https://github.com/open-telemetry/opentelemetry-demo-webstore/pull/143))
-* Added manual instrumentation to Java ad service
+* Added manual instrumentation to ad service
 ([#150](https://github.com/open-telemetry/opentelemetry-demo-webstore/pull/150))
 * Added manual instrumentation to Ruby email service
```

Comment:
Following that change on L26, is the intention to remove "Ruby" here as well?

Reply:
Removed